README.md

```diff
@@ -1,2 +1,37 @@
-# odh-manifests
-A repository for Open Data Hub Kustomize manifests extending upstream Kubeflow manifests
+# Open Data Hub Manifests
+A repository for [Open Data Hub](https://opendatahub.io) Kustomize manifests extending upstream Kubeflow manifests.
+
+## Community
+
+* Website: https://opendatahub.io
+* Documentation: https://opendatahub.io/docs.html (applicable for version 0.5.1)
+* Mailing lists: https://opendatahub.io/community.html
+* Community meetings: https://gitlab.com/opendatahub/opendatahub-community
+
+## Components
+
+Open Data Hub provides various AI/ML and Data Science tools on top of OpenShift Container Platform. The components currently available are:
+
```

Comment:
Maybe re-word or add. "Open Data Hub is an end to end AI/ML platform that provides various tools for Data Scientists and Engineers